src/pages/student/categories/CategoryDetail/components/SubcategoryCard/index.js

```diff
@@ -4,42 +4,39 @@ import Card from 'react-bootstrap/Card';
 import { PropTypes } from 'prop-types';
 import style from './index.module.css';
 
-const SubcategoryCard = ({title, firstRow, secRow, thirRow}) => {
-
+const SubcategoryCard = ({ category }) => {
   return (
     <Card className={style.card}>
-      <Card.Header id={style.cardHeader}>{title}</Card.Header >
-      <Card.Body >
-        <Link >
+      <Card.Header id={style.cardHeader}>{category?.name}</Card.Header>
+      <Card.Body>
+        <Link
+          to={
+            category?.subcategories_count
+              ? `/categories/${category.id}/sub`
+              : `/categories/${category.id}/quizzes`
+          }
+        >
           <div className={style.cardContent}>
-            <table style={{width: '100%'}}>
+            <table style={{ width: '100%' }}>
               <tr>
```

Comment:
enclose to a <tbody> tag to prevent console errors:
```
<tbody>
                <tr>
                  <td className={style.forSeccolum}>{category?.description}</td>
                </tr>
                <tr>
                  <td id={style.listTable}>Available Quizzes:</td>
                  <td className={style.forSeccolum}>
                    {category?.total_quizzes ?? '0'}
                  </td>
                </tr>
              </tbody>
```